africanlii/views/judgment.py

```diff
@@ -17,7 +20,7 @@ def get_queryset(self):
 
 
 @registry.register_doc_type("judgment")
-class JudgmentDetailView(DetailView):
+class JudgmentDetailView(DocumentVersionsMixin, DetailView):
```

Comment:
It's starting to look like a common `DocumentDetailView` base class would be useful that would be used by all document detail views.

Reply:
I had this at some point 😅